default_warehouse_from_sale_team/models/sales_team.py

```diff
@@ -31,13 +31,12 @@ def default_get(self, fields_list):
         """Force that if model has a field called warehouse_id the default
         value is the one in the sales team in the user setted
         """
-        defaults = super(WarehouseDefault,
-                         self).default_get(fields_list)
+        defaults = super(WarehouseDefault, self).default_get(fields_list)
         res_users_obj = self.env['res.users']
         user_brw = res_users_obj.browse(self._uid)
-        warehouse = user_brw.sale_team_id.default_warehouse_id
-        if warehouse:
-            warehouse_id = warehouse.id
+        allowed_warehouse_ids = user_brw.sale_team_ids.mapped('default_warehouse_id').sorted(lambda w: w.sequence)
```

Comment:
This may be simplified a bit:

```suggestion
        allowed_warehouse_ids = user_brw.sale_team_ids.default_warehouse_id.sorted('sequence')
```:

Reply:
You didn't remove the mapped. The `mapped` method is not required anymore for relational fields since v13 (more info on https://github.com/odoo/odoo/pull/32897)

Reply:
- [x] remove mapped

done @luisg123v